src/protected/application/lib/modules/Diligence/Controllers/Refo.php

```diff
@@ -30,39 +27,61 @@ function GET_report()
         $app->view->regObject['diligence'] = $dili;
         $app->view->regObject['registration'] = $reg;
         $mpdf = self::mpdfConfig();
-        self::mdfBodyMulti($mpdf,
-        'refo/report-finance', 
-        'Secult/CE - Relatório Financeiro',
-        'Diligence/assets/css/diligence/multi.css');
+        self::mdfBodyMulti(
+            $mpdf,
+            'refo/report-finance',
+            'Secult/CE - Relatório Financeiro',
+            'Diligence/assets/css/diligence/multi.css'
+        );
+    }
+
+    public function POST_deleteFinancialReport()
+    {
+        $app = App::i();
+        $conn = $app->em->getConnection();
+
+        $file = $app->repo('File')->findBy(['id' => (int) $this->data['fileId']])[0];
```

Comment:
Acredito que utilizar apenas o `find` seria mais adequado.
```suggestion
        $file = $app->repo('File')->find((int) $this->data['fileId']);
```